config/src/main/java/org/hyperledger/besu/config/GenesisConfigOptions.java

```diff
@@ -46,8 +46,12 @@
 
   OptionalLong getTangerineWhistleBlockNumber();
 
+  OptionalLong getEcip1015BlockNumber();
```

Comment:
ETC's SEO is problematic, so it is often hard to figure out what the actual ECIP texts are.  Could you add a good link to what each of these ETC options are?  Ideally to the ECIP that specifies it and in the "canonical" ECIP location?  I can do mainnet comments in another PR.